src/python/grpcio_health_checking/commands.py

```diff
@@ -43,7 +43,7 @@
 
 ROOT_DIR = os.path.abspath(os.path.dirname(os.path.abspath(__file__)))
 HEALTH_PROTO = os.path.abspath(
-    ROOT_DIR + '../../../proto/grpc/health/v1/health.proto')
+    os.path.join(ROOT_DIR, '../../../proto/grpc/health/v1/health.proto'))
```

Comment:
This ends up with `HEALTH_PROTO = $REPO_ROOT/proto/grpc/health/v1/health.proto`; one too many up-dirs.